### PR TITLE
Merge 1.7.1 into 1.7 branch.

### PR DESCRIPTION
**Pull request by rhass**
## Description
Merge 1.7.1 goreleser fix and release changes into 1.7 branch.

## Related Issues
n/a

## Motivation and Context
Cleanup the mess I made.

## How Has This Been Tested?
n/a

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improves code quality but has no user-facing effect)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Readiness Checklist

- [ ] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by alaudazzi — APPROVED** [review]
LGTM

File: docs/release_notes/v1.7.1.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+https://download.elastic.co/downloads/ecctl/1.7.1/ecctl_1.7.1_linux_386.tar.gz[ecctl_1.7.1_linux_386.tar.gz]
+https://download.elastic.co/downloads/ecctl/1.7.1/ecctl_1.7.1_linux_64-bit.deb[ecctl_1.7.1_linux_64-bit.deb]
+https://download.elastic.co/downloads/ecctl/1.7.1/ecctl_1.7.1_linux_64-bit.rpm[ecctl_1.7.1_linux_64-bit.rpm]
+https://download.elastic.co/downloads/ecctl/1.7.1/ecctl_1.7.1_linux_amd64.tar.gz[ecctl_1.7.1_linux_amd64.tar.gz]
+
+[float]
+[id="{p}-release-notes-v1.7.1-whats-new"]
+==== What's new
+
+// TODO: FILL OR REMOVE
```
Comment (obierlaire) [inline docs/release_notes/v1.7.1.adoc:28]: I think you need to actually "fill or remove" this :) Maybe remove this

File: docs/release_notes/v1.7.1.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+[float]
+[id="{p}-release-notes-v1.7.1-bug-fixes"]
+==== Bug fixes
+
+// TODO: FILL OR REMOVE
```
Comment (obierlaire) [inline docs/release_notes/v1.7.1.adoc:34]: I "think" this is where you should tell what we are doing here (it's kinda bugfix? )

File: docs/release_notes/v1.7.1.adoc [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+[float]
+[id="{p}-release-notes-v1.7.1-breaking-changes"]
+==== Breaking changes
+
+// TODO: FILL OR REMOVE
```
Comment (obierlaire) [inline docs/release_notes/v1.7.1.adoc:40]: Remove this section

File: docs/release_notes/v1.7.1.adoc [hunk 4]
```diff
@@ -0,0 +1,53 @@
+[id="{p}-release-notes-v1.7.1"]
+=== Elastic Cloud Control (ecctl) v1.7.1
+++++
+<titleabbrev>v1.7.1</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.7.1-whats-new,What's new>> | <<{p}-release-notes-v1.7.1-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.7.1-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.7.1-changelog,Changelog>>
+
+Welcome to the v1.7.1 release of {n}. This version brings new features, some breaking changes, and bug fixes.
```
Comment (obierlaire) [inline docs/release_notes/v1.7.1.adoc:9]:
```suggestion
Welcome to the v1.7.1 release of {n}. This version brings a bug fix.
```
Feel free to adjust this sentence explaining the purpose of this release

**Review by obierlaire — CHANGES_REQUESTED** [review]

File: docs/release_notes/v1.7.1.adoc (hunk 1 above)
Comment (rhass) [reply docs/release_notes/v1.7.1.adoc:28]: Hah! I was juggling a bunch of stuff when I made this and completely missed this. Thanks!

File: docs/release_notes/v1.7.1.adoc (hunk 4 above)
Comment (obierlaire) [reply docs/release_notes/v1.7.1.adoc:9]: I think you just edited my suggestion but did not committed it 😉

**Review by obierlaire — APPROVED** [review]

File: docs/release_notes/v1.7.1.adoc [hunk 5]
```diff
@@ -0,0 +1,35 @@
+[id="{p}-release-notes-v1.7.1"]
+=== Elastic Cloud Control (ecctl) v1.7.1
+++++
+<titleabbrev>v1.7.1</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.7.1-whats-new,What's new>> | <<{p}-release-notes-v1.7.1-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.7.1-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.7.1-changelog,Changelog>>
```
Comment (obierlaire) [inline docs/release_notes/v1.7.1.adoc:7]:
```suggestion
<<{p}-release-notes-v1.7.1-changelog,Changelog>>
```
Other refs does not exist, hence the error

**Pull request merged**